numba/cpython/builtins.py

```diff
@@ -573,7 +573,11 @@ def min_impl(indval1, indval2):
                     return indval2
             return indval1
         return min_impl
-    elif isinstance(indval1, types.scalars.Boolean) and \
+
+
+@overload(min)
+def indval_min(indval1, indval2):
```

Comment:
I'd suggest naming the function and variables to reflect the fact they are for booleans, and not `IndexValue` pairs, e.g.
```suggestion
def bool_min(bool1, bool2):
```

etc.